### PR TITLE
React assignment done

### PR DESCRIPTION
**Pull request by Brady76**
I only changed things from line 229 and down. I can explain whatever doesn't make sense. It didn't work before because I hadn't made a container for the cards in the Page section so I gave it a cell of column size 12 and added mdl-grid when Alex said he wanted it oriented like a grid. If you want any info on the grid it's under the layout section of the material design lite website.

### CODE REVIEW

**Comment by CarlosHL** [comment]
+1, looks cool, I am using your code for the add button!

File: src/templates/BradyModifiedDashboard.html [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+      </svg>
+    <script src="../assets/javascripts/browser.js"></script>
+    <script src="../assets/javascripts/react.js"></script>
+    <script src="../assets/javascripts/react-dom.js"></script>
+    <script src="../assets/javascripts/material.js"></script>
+  </body>
+</html>
+
+
+<script type="text/babel">
```
Comment (CarlosHL) [inline src/templates/BradyModifiedDashboard.html]: The cards are not rendering right, there are a little bit to the right.

**Comment by nicot** [comment]
Brady, do you still want to merge this?

**Comment by Brady76** [comment]
Nah it's alright, it was more so Carlos Alex and Michael could get a look at it

**Pull request closed without merging**